Slack.Automation/Promact.Core.Repository/EnvironmentVariableStore.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace Promact.Core.Repository
+{
+    public static class EnvironmentVariableStore
+    {
+        /// <summary>
+        /// This method is used to fetch the value of the environment variable of the given name
+        /// </summary>
+        /// <param name="VarName"></param>
+        /// <returns></returns>
+        public static string GetEnvironmentVariableValues(string VarName)
+        {
+            return Environment.GetEnvironmentVariable(VarName, EnvironmentVariableTarget.Process);
```

Comment:
Sir, I did not understand "I have to write a lot of code for this". Please elaborate which code you are referring to ?